modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/view/FDSView.java

```diff
@@ -38,6 +39,8 @@ public default FDSTableSchema getFDSTableSchema(Locale locale) {
 
 	public String getName();
 
+	public Map<String, Object> getOptions();
+
```

Comment:
I like the `options` property! Normal properties are serialized for each view, but inline actions only make sense for a table view. This is a good way to enable customization scoped to a view, while keeping the serializer simple.

The only improvement here can be the default value in interface, see for example the recently added [preloaded filters data](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-api/src/main/java/com/liferay/frontend/data/set/filter/FDSFilter.java#L28-L30). With this, `Base*View` classes don't need to be modified.

Reply:
Nice! I was wondering how to do this